src/components/AuthLayout/index.js

```diff
@@ -88,7 +78,9 @@ class AuthLayout extends Component {
           {backOpt === 'true' ? back : <Fragment />}
           <br />
           {heading === undefined ? (heading = 'login with...') : heading}
-          <Dropdown
+
```

Comment:
Not convinced it is worth keeping code that is commented out.  Perhaps file an issue to track what you'd need to do later for localization?

Reply:
removed and will be tracked with #115